src/objective-c/change-comments.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
```

Comment:
Michael - If someone ever sends a PR with //-style doc comments, it's best to have the script around.